docs/source/developers/release.rst

```diff
@@ -67,7 +70,8 @@ generated properly.
 Before creating a Release Candidate
 ===================================
 
-Ensure local tags are removed, gpg-agent is set and JIRA tickets are correctly assigned.
+Ensure local tags are removed, gpg-agent is set and GitHub issues and pull
+requests are correctly assigned.
```

Comment:
same as above, PRs at the moment are not required to be assigned.
I agree that this could be modified in the future as we might want to change the workflow, i.e. there has been discussions in the past of whether we require an issue always and we could just create PRs but at the moment, the PR milestone is not used on the scripts.

Reply:
Thanks for the catch here. I'll remove this bit.

Reply:
Changed in bd3cfd190fe72d69ee48ba3ef8e8c5ddd41b15f1.